sale_cancel_reason/__openerp__.py

```diff
@@ -19,15 +20,17 @@
 #
 #
 
-{'name': 'Sale Cancel Reason',
- 'version': '8.0.1.1',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
- 'category': 'Sale',
- 'license': 'AGPL-3',
- 'complexity': 'normal',
- 'images': [],
- 'website': "http://www.camptocamp.com",
- 'description': """
+{
+    'name': 'Sale Cancel Reason',
+    'version': '9.0.1.0.0',
+    'author': 'Camptocamp, Odoo Community Association (OCA),'
+            ', Serpent Consulting Services Pvt. Ltd.',
```

Comment:
In this case, it's correct to be added here, as there's no contributor key on the manifest, and also contributors can expand more the module than the original author, but without extra comma (there are 2).